src/devices/ExplorerHat/Lights.cs

```diff
@@ -104,24 +106,22 @@ public void Off()
 
         #region IDisposable Support
 
-        private bool _disposedValue = false; // Para detectar llamadas redundantes
+        private bool _shouldDispose;
```

Comment:
The pourpuse of the previous line was different though, it was meant to make sure that the double call to Dispose() wouldn't crash the device